lib/matplotlib/collections.py

```diff
@@ -394,7 +394,7 @@ def set_hatch(self, hatch):
         *hatch* can be one of::
 
           /   - diagonal hatching
-          \   - back diagonal
+          \\   - back diagonal
```

Comment:
Done. I can't rebuild matplotlib today apparently, so can't test the html versions.